### PR TITLE
Fixing bug 1282

### PR DESCRIPTION
**Pull request by igorsteinmacher**
Duplicate PR #59

### CODE REVIEW

**Comment by koppor** [comment]
Is this a clean version of #59? Should we close #59 and concentrate on this one?

**Comment by koppor** [comment]
The thing is, we want to improve our code quality. Could you use a formatter as described in CONTRIBUTING.md? May I also ask to remove commit a6919e6 from here as it has nothing to do with this PR? You can open a separate PR for updating the translation :). Sorry for being picky, but we are really trying to improve our code quality.

**Comment by igorsteinmacher** [comment]
OK Oliver,
I will format the code accordingly and remove the commit from the PR

--
Igor Fabio Steinmacher, Ph.D.
Professor in Dept. of Computing at Universidade Tecnológica Federal do
Paraná

**Comment by simonharrer** [comment]
Looks good to me.

File: src/main/java/net/sf/jabref/sql/exporter/DBExporter.java [hunk 1]
```diff
@@ -131,10 +131,12 @@ private void populateEntriesTable(int database_id,
                 query = query + ", ";
                 val = entry.getField(SQLUtil.getAllFields().get(i));
                 if (val != null) {
-                    val = val.replace("\\", "\\\\");
-                    val = val.replace("\"", "\\\"");
-                    val = val.replace("\'", "''");
-                    val = val.replace("`", "\\`");
+                    if(dbStrings.getServerType().equals("MySQL")){
```
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/sql/exporter/DBExporter.java]: I would love to see a comment here why this is necessary. So future developers can instantly see the reason behind this code and remove it if behaviour has changed.

**Comment by koppor** [comment]
Last thing: Could you add a CHANGELOG entry? :innocent:

**Comment by igorsteinmacher** [comment]
Done it.
OK! A long time I didn't send anything myself. I agree that I need to follow the process and practices, and I did not read them after returning... Hope this is good to go @koppor

**Pull request merged**